### PR TITLE
Fix rule hostname typos (#12297)

### PR DESCRIPTION
**Pull request by jayvdb**
The simplest subset of #12297 hosts in regex that are not in targets.

### CODE REVIEW

**Comment by jayvdb** [comment]
Fetch failures:
- https://icmail.net/ - This appears to be the intended target, but maybe non-functional now .  http://icmail.com/ (not https) is now a parked domain, and it was a parked domain for the few dates I checked on archive.org
- http://cdn.therepublic.com/ - fails to resolve; http://cdn1.therepublic.com/ is working; will update ruleset
- https://ww2.epeat.com/ - timeout; http://ww2.epeat.net is working; will update ruleset
- http://orionmagazine.com/ - timeout; https://orionmagazine.org/ is working; will update ruleset
- http://research-store.com/ - timeout; http://researchstore.com/ is parked domain now

**Comment by jayvdb** [comment]
research-store.com and icmail moved to https://github.com/EFForg/https-everywhere/pull/18949

File: src/chrome/content/rules/Orion_Magazine.org.xml [hunk 1]
```diff
@@ -2,9 +2,17 @@
 
 	<target host="orionmagazine.org" />
 	<target host="www.orionmagazine.org" />
+	<target host="orionmagazine.com" />
```
Comment (jayvdb) [inline src/chrome/content/rules/Orion_Magazine.org.xml]: Moved to https://github.com/EFForg/https-everywhere/pull/18952

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**